doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -122,6 +122,13 @@ given to L<OSSL_STORE_open_file(3)>, the returned name will be the path of
 each object, so if C</foo/bar> was given and that path has the file
 C<cookie.pem>, the name C</foo/bar/cookie.pem> will be returned.
 
+Each loader is responsible for ensuring that it only returns unique
+and permanent URIs for the storage where the object (or collection of
+objects) resides.
+However, it's possible that certain schemes allow an object (or collection
+thereof) to be reached with alternative URIs; just because one URI is
+canonical doesn't mean that other variants can't be used.
```

Comment:
Yeah, the trouble I had with your variant is that it became ugly, doubling the information.  So how about this instead (whole paragraph shown):

```
The returned URI is considered canonical and must be unique and permanent
for the storage where the object (or collection of objects) resides.
Each loader is responsible for ensuring that it only returns canonical
URIs.
However, it's possible that certain schemes allow an object (or collection
thereof) to be reached with alternative URIs; just because one URI is
canonical doesn't mean that other variants can't be used.
```